presto-main/src/main/java/com/facebook/presto/operator/DistinctLimitOperator.java

```diff
@@ -51,6 +51,7 @@
         private final Optional<Integer> hashChannel;
         private boolean closed;
         private final JoinCompiler joinCompiler;
+        private final int timeoutMillis;
```

Comment:
Should we use duration for this?

Reply:
Not really because we are checking millis and lets user know these should be small values. SO this more descrioptive.

Reply:
Also note that we don't expect users to change it normally except in rare cases as setting it to high number can slow down their query